templates/single-course/modules.php

```diff
@@ -94,7 +94,18 @@
 
 						<header>
 
-							<h3><?php esc_html_e( 'Lessons', 'sensei-lms' ); ?></h3>
+							<h3>
+								<?php
+								$course_id    = Sensei()->lesson->get_course_id( get_the_ID() );
```

Comment:
This actually doesn't work because `get_the_ID` is sometimes the course ID and other times its the lesson ID.

Besides that, in this case we need to find the number of lessons in a _module_, not the number of lessons in a _course_.

In `template-functions.php`, there is a "Module specific template tags" section. I think it's probably easiest to add a new function in that section that gets a count of the lessons in a module, and then call it from `modules.php`:

```
function sensei_module_lesson_count() {
	global $sensei_modules_loop;

	$course_id      = $sensei_modules_loop['course_id'];
	$module_term_id = $sensei_modules_loop['current_module']->term_id;
	$module_lessons = Sensei()->modules->get_lessons( $course_id, $module_term_id );

	return count( $module_lessons );
}
```

Reply:
Ok. I will follow that way. Do I need to add `isset()` checks for `$course_id` and `$module_term_id` in that function?

Reply:
> Ok. I will follow that way. Do I need to add `isset()` checks for `$course_id` and `$module_term_id` in that function?

Check where they're being set and decide if it's safe or not.